gradle.properties

```diff
@@ -10,7 +10,7 @@ cordaProductVersion = 5.1.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 1
+cordaApiRevision = 2
```

Comment:
I think this PR breaks the API because the interface `TokenSelection` now has an extra function that needs to be overriden. Old code will break because the derived class won't have the implementation for `TokenBalance queryBalance(@NotNull TokenBalanceCriteria criteria)`.